docs/src/howto-guides/008_config_local_mqtt_port.md

```diff
@@ -34,14 +28,26 @@ tedge config get mqtt.port
 ```
 This prints out the port that has been set.
 
+## Step 3: Connect the thin edge device to cloud
+
+Using `tedge` command connect to the desired cloud as below.
+
+This will force all the services (mosquitto, tedge_mappers) to use newer port.
+
+```shell
+tedge connect c8y/az
```

Comment:
This line is a little bit confusing, I would rather you use specific command intended because when someone copy paste this one it will fail.
```suggestion
tedge connect c8y
# or
tedge connect az
```

Reply:
fixed